src/modules/tenant/actions.js

```diff
@@ -1,4 +1,3 @@
 export const types = {
-	TENANTS_CHANGED: "asab/TenantService/CHANGED",
-	TENANT_SELECTED: "asab/TenantService/SELECT_TENANT",
+	AUTH_USERINFO: "SeaCat/AUTH_USERINFO",
```

Comment:
auth should stay in the auth service  tenant in the tenant service ... mixing it si 100% wrong way.